app/src/main/res/values/strings.xml

```diff
@@ -396,6 +396,17 @@
     <string name="settingsAiEngineDescription">Select your AI writing assistant</string>
     <string name="settingsAppLock">App lock</string>
     <string name="settingsAuthorizeTracking">Authorize tracking</string>
+    <string name="settingsAutoAdvanceDescription">Select what to display when a message is archived or deleted</string>
+    <string name="settingsAutoAdvanceFollowingThreadDescription">Display following thread when archiving or deleting</string>
+    <string name="settingsAutoAdvanceFollowingThreadTitle">Following thread</string>
+    <string name="settingsAutoAdvanceListOfThreadsDescription">Display list of threads when archiving or deleting</string>
+    <string name="settingsAutoAdvanceListOfThreadsTitle">List of threads</string>
+    <string name="settingsAutoAdvanceNaturalThreadDescription">Repeats the last action you performed</string>
+    <string name="settingsAutoAdvanceNaturalThreadHint">Automatically selects the next or previous thread, depending on your last interaction.</string>
+    <string name="settingsAutoAdvanceNaturalThreadTitle">Repeat last action</string>
+    <string name="settingsAutoAdvancePreviousThreadDescription">Display previous thread when archiving or deleting</string>
+    <string name="settingsAutoAdvancePreviousThreadTitle">Previous thread</string>
+    <string name="settingsAutoAdvanceTitle">Auto advance</string>
```

Comment:
Archiving, deletion, but also moving Message/Thread.
Are we modifying the Locos strings to take this into account ?
Has iOS also implemented the feature when moving ?

Reply:
Yes they implemented the feature when moving

Reply:
The question was asked to the UI team, waiting for their answer.

Reply:
The decision was taken to merge as is, and to update the wording later.